src/controllers/forgotpassword.js

```diff
@@ -12,7 +12,7 @@ define([], function() {
         });
         if ("PinCode" == result.Action) {
             var msg = Globalize.translate("MessageForgotPasswordFileCreated");
-            return msg += "<br/>", msg += "<br/>", msg += result.PinFile, msg += "<br/>", void Dashboard.alert({
+            return msg += "<br/>", msg += "<br/>", msg += "<a href=\"/web/index.html#!/forgotpasswordpin.html\">Enter PIN here to finish Password Reset</a>" ,msg += "<br/>",msg += result.PinFile, msg += "<br/>", void Dashboard.alert({
```

Comment:
please **nooo**, this leading slash in a URL kills all ability to run this prefixed... there should be some `getBaseUrl()` or something.

Reply:
Just adding the page itself as `forgotpasswordpin.html` will work fine.

Reply:
Switched it to just the html file, that sounds good.